sdk/monitor/azure-monitor-ingestion/src/main/java/com/azure/monitor/ingestion/models/LogsUploadOptions.java

```diff
@@ -17,6 +17,14 @@ public final class LogsUploadOptions {
     private Integer maxConcurrency;
     private Consumer<LogsUploadError> logsUploadErrorConsumer;
 
+    // empty constructor to suppress javadoc warning
```

Comment:
I had to add the default constructor due to Java 21's warnings against constructors without javadocs.

Reply:
Remove this line.